src/packages/someotherstructure/server/some-other-scripts-name.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
I'd this file necessary?

Reply:
If these files are necessary, let's add a comment to them indicating why they exist 👍

Reply:
nope, sorry they arent needed any more